phpunit/html-api/class-gutenberg-html-tag-processor-6-3-test.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
we do have a `_bookmark` test file where we're holding bookmarking and seeking behaviors. the main test file is pretty big, and although this is small, it might make sense to keep that separation so that when we merge back into Core it will be easier.

Reply:
Renamed to include the `_bookmark` suffix (well, infix) in https://github.com/WordPress/gutenberg/pull/47559/commits/f82551d790cbb8d67e804962b1f4585d6b20418b.

(Note that the test file naming convention in GB still diverges from Core's.)

Reply:
> test file naming convention in GB still diverges from Core's

and both diverge from `PHPUnit` :sigh: